docs/1-Applying/finding_companies_to_apply_to.md

```diff
@@ -4,30 +4,36 @@
 
 You can often find lists online of companies hiring for summer internships. Internships at other times of the year can be trickier, but you can always email companies that do summer internships and ask if they'll hire at other times of the year.
 
-- [http://www.intern.supply/](http://www.intern.supply/)
-- https://www.facebook.com/careers/university/?refid=8&__tn__=R
-- Research: https://www.nsf.gov/crssprgm/reu/list_result.jsp?unitid=5049
-- https://www.indeed.com/
-- https://breakoutlist.com/all
-- https://www.wayup.com/
+- [Intern Supply](http://www.intern.supply/) (links you directly to the application page of a lot of companies)
+- [Research in the US](https://www.nsf.gov/crssprgm/reu/list_result.jsp?unitid=5049)
+- [Indeed.com](https://www.indeed.com/) (job board)
+- [Breakout List](https://breakoutlist.com/all) (hot new tech companies)
+- [WayUp](https://www.wayup.com/) (internships)
+- [Hacker News](https://news.ycombinator.com/) (for the latest monthly list, search for "Ask HN: Who is hiring? (<month> <year>)". For example, "Ask HN: Who is hiring? (February 2017)".)
 - TODO: add some links to lists people often use (idk, can some people recommend? I haven't used these much)
 
 ### Join Facebook groups (and mailing lists, slack channels, whatever is out there) that have job postings.
 
-- [https://www.facebook.com/groups/HHinternships/?ref=br_rs](https://www.facebook.com/groups/HHinternships/?ref=br_rs)
-- [https://www.facebook.com/groups/733654363389653/?ref=br_rs](https://www.facebook.com/groups/733654363389653/?ref=br_rs)
-- [https://www.facebook.com/groups/ncwandintern/?ref=br_rs](https://www.facebook.com/groups/ncwandintern/?ref=br_rs)
+- [Hackathon Hackers Internships](https://www.facebook.com/groups/HHinternships)
+- [Hackathon Hackers Internships/Co-ops](https://www.facebook.com/groups/733654363389653)
+- [Bay Area Jobs Network](https://www.facebook.com/groups/ncwandintern)
 - etc... you should also check out intern groups relevant to your school! they probably exist
 
-
 ### Google it
 
-Sometimes I just google "hiring interns" (usually adding a preferred location and/or an industry I'd like to work in) and see what shows up! This works pretty well too
+Information about available opportunities is scattered throughout the Internet moves around a lot. Sometimes I just google "hiring interns" (adding a few more specific terms) and see what shows up! This works pretty well too.
+
+The more specific the query, the more likely you can find something useful. Try different combinations of queries that include some of the below:
+- What you're looking for. It can be "tech company internships", "hiring programming intern", etc. Since not all companies have internship positions advertised, you may want to just search for "tech company" and manually find someone to contact on their website.
```

Comment:
"manually find someone to contact on their website." => "look around on their website for someone to ask about internships"

(not necessarily those words, but I think 'manually' isn't super descriptive)